examples/resolver.rs

```diff
@@ -1,15 +1,18 @@
+///! See documentation at <https://docs.rs/oxc_resolver>
 use std::{env, path::PathBuf};
 
 use oxc_resolver::{AliasValue, ResolveOptions, Resolver};
 
 fn main() {
-    // Path to directory, must be in absolute path.
-    let path = env::args().nth(1).expect("path");
+    let path = PathBuf::from(env::args().nth(1).expect("path"));
+
+    assert!(path.is_dir(), "{path:?} must be a directory that will be resolved against.");
```

Comment:
Is it safe to say that `path` is the same path that `package.json` is? At least, in the majority of the cases. If so, maybe we should mention it. Saying path is still out of context, and maybe mention  other things like `package.json` could help.

Reply:
Nope, it's `__dirname` or `import.meta.url` - I added relevant doc to the `resolve` method.